pulibrary/orangelight
# Flipflop with admins

To verify this PR 
- Go to catalog-staging (assuming this branch is still deployed)
- log in, and notice if there's a banner with "Flipflop test_header feature is on!" is at the top of the page
- Go to https://catalog-staging.princeton.edu/features and flip the feature the other way (if you can see this page, it means you are an admin)
- Go to the home page and see if the banner is there - it should be the opposite of what it was. If it's not, try a hard refresh. 

Closes #3132 
Closes #3130
